change/@ni-nimble-components-568f1dc7-fe9b-4288-991a-c0d99cb4e256.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

No new APIs, just a bug fix, so I'd vote "patch"
```suggestion
  "type": "patch",
```